packages/rn-tester/js/RNTesterAppShared.js

```diff
@@ -265,40 +266,42 @@ const RNTesterApp = ({
     screen === Screens.COMPONENTS ? examplesList.components : examplesList.apis;
 
   return (
-    <RNTesterThemeContext.Provider value={theme}>
-      <RNTTitleBar
-        title={title}
-        theme={theme}
-        documentationURL={activeModule?.documentationURL}>
-        {activeModule && BackButtonComponent ? (
-          <BackButtonComponent onBack={handleBackPress} />
-        ) : undefined}
-      </RNTTitleBar>
-      <View
-        style={StyleSheet.compose(styles.container, {
-          backgroundColor: theme.GroupedBackgroundColor,
-        })}>
-        {activeModule != null ? (
-          <RNTesterModuleContainer
-            module={activeModule}
-            example={activeModuleExample}
-            onExampleCardPress={handleModuleExampleCardPress}
+    <ReportFullyDrawnView style={styles.container}>
```

Comment:
@janicduplessis this is causing all of our internal E2E tests to fail. Can we find a different approach that doesn't rely on us having to modify the root of the app layout?